docs/design/federation-phase-1.md

```diff
@@ -140,14 +140,25 @@ time.
 
 ## Ubernetes Scheduler
 
-The Ubernetes Scheduler schedules resources onto the underlying
-Kubernetes clusters.  For example it watches for unscheduled Ubernetes
-replication controllers (those that have not yet been scheduled onto
-underlying Kubernetes clusters) and performs the global scheduling
-work.  For each unscheduled replication controller, it calls policy
-engine to decide how to spit workloads among clusters. It creates a
-Kubernetes Replication Controller on one ore more underlying cluster,
-and post them back to `etcd` storage.
+Whereas the Kubernetes scheduler schedules pending pods onto
+underlying nodes, the Ubernetes Scheduler schedules resources onto the
+underlying Kubernetes clusters.  So the Ubernetes scheduler creates
+per-cluster Kubernetes API objects corresponding to Ubernetes API
+objects.
+
+For example, for Replication Controllers:
+
+1. watches it's persistent store (typically etcd) for unscheduled
```

Comment:
To be clear, the controller should not talk to etcd directly. It should go through the api server.